phpmyfaq/translations/language_pl.php

```diff
@@ -1,847 +1,1438 @@
 <?php
 
 /**
- * Polish language file
+ * The English language file - try to be the best of British and American English
```

Comment:
This change has to be reverted

Reply:
Wenn Sie freundlich wären, und wenn du es korrigieren könntest, wäre es schön. Ich würde lieber nicht zerstöre nichts. Bitte, Please.